autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -21,18 +21,3 @@ def get_all_terms(year: int = -1) -> List[str]:
 
     return [f"{year}{semester}{location}"
             for year, semester, location in product(years, semesters, locations)]
-
-
-def slice_every(iterable: Iterable, n: int) -> Iterable[Iterable]:

```

Comment:
rest in peace my sweet child

Reply:
![gif](https://media.giphy.com/media/3oFzmrqh43AvYwn9Cw/giphy.gif)